08-wt-gradebook.Rmd

```diff
@@ -1,12 +1,30 @@
 # Walkthrough 2: Approaching Gradebook Data From a Data Science Perspective {#c08}
 
+## Vocabulary
+
+- directory  
+- environment    
+- pivot  
+- linearity  
+- correlation  
+- outliers  
+- linear model  
+
 ## Introduction
 
 There are a variety of data sources to explore in the education field. Student assessment scores can be examined for progress towards goals. The text from a teacher’s written classroom observation notes about a particular learner’s in-class behavior or emotional status can be analyzed for trends. We can tap into the exportable data available from common learning software or platforms popular in the K-12 education space.
 
-## Data Sources
+### Background
+
+This walkthrough goes through a series of analyses using the data science framework. The first analysis centers around a ubiquitous K-12 classroom tool: the gradebook. While gradebook data is nearly ubiquitious in education, it is sometimes ignored in favor of data collected by evaluators and researchers or data from state-wide tests. Nevertheless, it represents an important, occasionally untapped data source, and one for which a data science approach can reveal the potential of the data science framework for analyzing a range of educational data sources.
```

Comment:
is this an okay addition?

Reply:
sorry for the late reply - GH notifications are getting split between spam and my inbox. this is good! I like that it highlights the ubiquitousness of gradebooks as an untapped resource!

Reply:
sweet